### PR TITLE
Supplementary figure PDF organization

### PR DESCRIPTION
**Pull request by sjspielman**
#### What scientific question is your analysis addressing?

From issue #1267, this PR adds more organization to the `figures/pdfs/supp` directory.

#### What was your approach?

+ Updated `figures/README.md` to explain this organization
+ Added two undocumented supplementary figure scripts to the `figures/scripts/README.md`
+ Updated relevant scripts to export figures to appropriate directory, and moved figures accordingly

#### Which areas should receive a particularly close look?

+ Should I change new directory names to, for example, `pdfs/supp/figS3` instead of  `pdfs/supp/fig3` to be even clearer these directories are supp?
+ There is one script in `figures/scripts/TelomeraseActivities.R` that exports some supplementary figures, but not clear which supplementary figure(s) or panel(s) these will go into. That said, it also doesn't appear as though this script has been run and actually generated figures in these directories. My sense is this script will be substantially updated (maybe entirely rewritten) when these figures are solidified, since the main text figures for telomerase activity have to be updated too. For now I am leaving this file alone, in favor of future issues which will deal with telomerase activities.
+ There is also the directory `pdfs/supp/sample-distribution/`. Should that be somewhere in particular?

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?

Yes

#### Reproducibility Checklist

- [ ] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [ ] This analysis has been added to continuous integration.

#### Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

File: figures/README.md [hunk 1]
```diff
@@ -72,9 +72,12 @@ This section includes details for file organization and guidelines that are not
 
 ### Individual panels and where to find them
 
-Individual panels, including any standalone legends, should be saved as **PDFs** in `figures/pdfs/<figure>/panels`.
+Individual panels, including any standalone legends, intended for the main text should be saved as **PDFs** in `figures/pdfs/<figure>/panels`.
 Each figure has its own subdirectory within `figures/pdfs`; individual panels for Figure 1 would be saved in `figures/pdfs/fig1/panels`, for Figure 2 in `figures/pdfs/fig2/panels` and so on and so forth.  
 
+Similarly, panels and standalone legends intended for the supplementary materials should be saved as **PDFs** in `figures/pdfs/supp/<figure>/panels`. 
+As for main text figures, individual panels for Figure S1 would be saved in `figures/pdfs/supp/figs1/panels`, and so on.
```
Comment (jaclyn-taroni) [inline figures/README.md:79]: Shouldn't all these changes be moving files to `figures/pdfs/supp/figs3`, `figures/pdfs/supp/figs5`, etc. then?

File: figures/README.md [hunk 2]
```diff
@@ -89,7 +92,8 @@ PNGs version can be linked in the accompanying manuscript repository [`AlexsLemo
 Each figure should include its own associated README in the appropriate PDF subdirectory, e.g., the documentation for Figure 4 can be found in `figures/pdfs/fig4/README.md`.
 That README should detail what analysis modules or individual scripts are required to generate the individual panels for that figure.
 
-Scripts that are exclusively for creating publication-ready plots are available in `figures/scripts` and should be documented in the `figures/scripts/README.md`.
+Scripts that are exclusively for creating publication-ready plots, both main text and supplementary, are available in `figures/scripts` and should be documented in the `figures/scripts/README.md`.
+
```
Comment (jaclyn-taroni) [inline figures/README.md:96]: Want to add something about using the `supp-` prefix for scripts? That appears to be the convention we are following.

File: figures/README.md (hunk 1 above)
Comment (sjspielman) [reply figures/README.md:79]: Yes, that was done here already. Did I miss something?

Comment (jaclyn-taroni) [reply figures/README.md:79]: Sorry, I could have been more specific. There is no `s` in the directory name, so we have `pdfs/supp/fig5/panels`, etc.


Screen Shot 2022-03-27 at 6 09 53 PM

Look I found a use for the new file tree!

Comment (sjspielman) [reply figures/README.md:79]: Aha, I missed your `s`. Yes, I should change this.

File: figures/README.md (hunk 2 above)
Comment (jaclyn-taroni) [reply figures/README.md:96]: > To me both is fine as long as it's documented in the README.

Agree

**Review by jaclyn-taroni — APPROVED** [review]
LGTM

**Pull request merged**